libbeat/outputs/logstash/json.go

```diff
@@ -34,14 +34,11 @@ func makeLogstashEventEncoder(beat string) (func(interface{}) ([]byte, error), e
 
 		buf.WriteRune('{')
 		if _, hasMeta := event["@metadata"]; !hasMeta {
-			typ := event["type"].(string)
-			buf.WriteString(`"@metadata":{"type":`)
-			encodeString(buf, typ)
+			buf.WriteString(`"@metadata":{"type":"doc","beat":`)
```

Comment:
Yeah, i think it's safer to hardcode the metadata one to `doc`, as otherwise existing Logstash configs will put it in `_type`.